src/App.tsx

```diff
@@ -50,8 +47,15 @@ class App extends React.Component<Props, State> {
     }))
   }
 
+  public setElection = (election: OptionalElection) => {
+    this.setState({ election })
+  }
+
   public render() {
-    const { contests } = this.props.election
+    if (!this.state.election) {
```

Comment:
this is the kind of conditional that I worry about once the election is nullable. We lose some of the typescript goodness.

Reply:
Yes, I agree. I want to make sure that the election object is not part of any component state. See previous comment for future solution.